cmd/skogul/main.go

```diff
@@ -743,9 +746,17 @@ func main() {
 		os.Exit(0)
 	}
 
-	c, err := config.File(*ffile)
-	if err != nil {
-		log.Fatal(err)
+	var c *config.Config
+
+	// Allow config file to be missing if running in -test mode,
+	// require it otherwise.
+	_, confFileErr := os.Stat(*ffile)
+	if !*ftest && os.IsNotExist(confFileErr) {
```

Comment:
Doesn't this mean that we will only try reading the config file if -test is missing AND the configfile is missing?